tools/cmd/runner/main.go

```diff
@@ -76,7 +76,8 @@ func main() {
 	log.Printf("Queue concurrency levels: %v", c)
 	log.Printf("Output directories: %v", outputDirMap)
 
-	r := runner.NewRunner(runner.NewLoadTestGetter(), runner.AfterIntervalFunction(p), retries, deleteSuccessfulTests)
+	grpcClientset := runner.NewGRPCTestClientSet()
```

Comment:
Here we are really want to create a LoadTestGetter and a PodsGetter. Perhaps it would be better to have the grpcClientset be separate from a k8sClientset and each of them to create their respective getter.

Reply:
PTAL at main.go now...is this what you mean?